packages/terra-detail-view/lib/_variables.scss

```diff
@@ -0,0 +1,11 @@
+// Variables
+// ======================
+
+$terra-detail-view-padding: 1.428rem !default;
+$terra-detail-view-footer-color: $terra-grey-50 !default;
+$terra-detail-view-subtitle-margin-bottom: 0.357rem !default;
+$terra-detail-view-subtitle-line-height: 1.428rem !default;
```

Comment:
When you add this to the CSS, use [unitless line-height](https://css-tricks.com/almanac/properties/l/line-height/#article-header-id-0). e.g. `line-height: 1.428;`